app/src/main/java/com/example/teamyuml/decentworkmobile/fragments/ListViewFragment.java

```diff
@@ -0,0 +1,216 @@
+                    startActivity(toWorkerDetail);
+                }
+            }
+        });
+    }
+
+    private void getNotice() {
+        final JsonObjectRequest jsonObjectRequest = new JsonObjectRequest(
+            Request.Method.GET, URL, null, new Response.Listener<JSONObject>() {
+            @Override
+            public void onResponse(JSONObject response) {
+                try {
+                    JSONArray results = response.getJSONArray("results");
+
+                    for (int i = 0; i < results.length(); i++) {
+                        JSONObject notice = results.getJSONObject(i);
+                        int id = notice.getInt("id");
+                        String title = notice.getString("title");
+                        String profession = notice.getString("profession");
+
+                        HashMap<String, String> oneNotice = new HashMap<>();
+                        oneNotice.put("id", String.valueOf(id));
+                        oneNotice.put("title", title);
+                        oneNotice.put("profession", profession);
+
+                        data.add(oneNotice);
+                    }
+                } catch (JSONException e) {
+                    e.printStackTrace();
+                }
+
+                if (data != null) {
+                    adapterClass = new CustomListView(getActivity(), data);
+                    initListView();
+                    Toast.makeText(getActivity(),
+                        "Pobrano " + data.size() + " ogłoszenia",
+                        Toast.LENGTH_LONG).show();
+                }
+            }
+        }, new Response.ErrorListener() {
+            @Override
+            public void onErrorResponse(VolleyError error) {
+                Toast.makeText(getActivity(),
+                    "Coś poszło nie tak",
+                    Toast.LENGTH_LONG).show();
+            }
+        });
+
+        VolleyInstance.getInstance(getActivity()).addToRequestQueue(jsonObjectRequest, "Notice");
+    }
+
+    private void getWorkers() {
+        final JsonArrayRequest jsonArrayRequest = new JsonArrayRequest(
+            Request.Method.GET, URL, null, new Response.Listener<JSONArray>() {
+            @Override
+            public void onResponse(JSONArray response) {
+                try {
+                    for (int i = 0; i < response.length(); i++) {
+                        JSONObject profile = response.getJSONObject(i);
+                        JSONObject user = profile.getJSONObject("user");
+                        String id = String.valueOf(user.getInt("id"));
+                        String name = user.getString("first_name");
+                        String last_name = user.getString("last_name");
+
+                        /*TODO: Professions is a list so we have to
+                          think how we want to store it in viewHolder*/
+                        JSONArray professionsJson = (JSONArray) profile.get("professions");
```

Comment:
If you have oneNotice.put("profession", profile.getString("professions")); you don't need this 5 lines at this point.